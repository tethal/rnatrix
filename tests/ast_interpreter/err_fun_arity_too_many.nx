fun add(a, b) {
    print(a + b);
}

fun main() {
    add(1, 2, 3);
}