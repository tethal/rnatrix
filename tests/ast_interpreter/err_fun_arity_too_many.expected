<string>:6:5: error: function add expects 2 arguments, but 3 were provided
    add(1, 2, 3);
    ^^^^^^^^^^^^
